### PR TITLE
Update/keyboard focus contrast

### PR DESCRIPTION
**Pull request by j-mnizhek**
- Update contrast for keyboard-focus
- Update border for invalid and valid states for Checkbox, Radio, Textarea, Input, InlineInput
- Update border for Tooltip with `warning` theme

### CODE REVIEW

**Comment by j-mnizhek** [comment]
These updates are necessary to make our focus states more accessible according to contrast checkers
image

File: semcore/checkbox/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [6.2.0] - 2022-10-27
+
+### Changed
+
+- Updated border-color in focus state from `--blue-400` to `--blue-500`, `--red-400` to `--red-500`, `--green-400` to `--green-500`.
```
Comment (ekrasnopolskaia) [inline semcore/checkbox/CHANGELOG.md:9]: just a little note about no green in the checkboxes

File: semcore/input/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,16 @@
...
 
+## [3.3.0] - 2022-10-27
+
+### Changed
+
+- Updated border-color in focus state from `--blue-400` to `--blue-500`, `--red-400` to `--red-500`, `--green-400` to `--green-500`.
+
 ## [3.2.3] - 2022-10-28
 
+## [3.2.2] - 2022-10-20
```
Comment (ekrasnopolskaia) [inline semcore/input/CHANGELOG.md:13]: I think we need to sort out the versions

File: semcore/radio/CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,16 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.2.0] - 2022-10-27
+
+### Changed
+
+- Updated border-color in focus state from `--blue-400` to `--blue-500`, `--red-400` to `--red-500`, `--green-400` to `--green-500`.
```
Comment (ekrasnopolskaia) [inline semcore/radio/CHANGELOG.md:9]: just a little note about no green here

**Review by ekrasnopolskaia — COMMENTED** [review]
Thank you for this pr!

**Pull request merged**